server/src/test/java/org/opensearch/action/admin/cluster/health/ClusterHealthRequestTests.java

```diff
@@ -75,14 +75,14 @@ public void testValidation() {
         ClusterHealthRequest clusterHealthRequest = randomRequest();
         {
             clusterHealthRequest.local(false);
-            clusterHealthRequest.ensureNodeCommissioned(true);
+            clusterHealthRequest.ensureNodeWeighedIn(true);
             ActionRequestValidationException e = clusterHealthRequest.validate();
             assertNotNull(e);
-            assertTrue(e.getMessage().contains("not a local request to ensure local node commissioned"));
+            assertTrue(e.getMessage().contains("not a local request to ensure local node commissioned or weighed in"));
         }
         {
             clusterHealthRequest.local(true);
-            clusterHealthRequest.ensureNodeCommissioned(false);
+            clusterHealthRequest.ensureNodeWeighedIn(false);
```

Comment:
Can we please also add an integ tests to cover all combinations of healthy/unhealthy/weighed in and decommissioned please

Reply:
Added

Decommission test is captured here https://github.com/opensearch-project/OpenSearch/blob/9dd36321626a930a8aef620451cd4cff6be5548e/server/src/internalClusterTest/java/org/opensearch/cluster/coordination/AwarenessAttributeDecommissionIT.java#L308